tests/test_integration.py

```diff
@@ -163,6 +163,7 @@ def test_sched_a_fulltext_trigger(self):
         )
         db.session.commit()
         manage.update_aggregates()
+        manage.refresh_itemized()
```

Comment:
It may not be needed - I did it as a safeguard since the code was originally being called as a part of `manage.update_aggregates()` but I can go back through and see if it's necessary in all of them.